magenta-lib/src/main/scala/magenta/deployment_type/FastlyCompute.scala

```diff
@@ -0,0 +1,102 @@
+package magenta.deployment_type
+
+import magenta.{DeploymentResources, KeyRing, Region}
+import magenta.deployment_type.AutoScaling.{
+  prefixApp,
+  prefixPackage,
+  prefixStack,
+  prefixStage
+}
+import magenta.tasks.{S3Upload, SSM, UpdateFastlyPackage}
+import magenta.tasks.{S3 => S3Tasks}
+import software.amazon.awssdk.services.s3.S3Client
+import software.amazon.awssdk.services.ssm.SsmClient
+
+object FastlyCompute extends DeploymentType with BucketParameters {
+  val name = "fastly-compute"
+
+  val documentation: String =
+    """
+      |Deploy a [Compute@Edge](https://www.fastly.com/products/edge-compute) package via the Fastly API.
+      |
+      |Example:
+      |
+      |```yaml
+      |your-service:
+      |  type: fastly-compute
+      |```
+      |Note that `your-service` must match the App name in Deployment Resources under `credentials:fastly`.
+      |It will appear there once it has been added to `prism-data.json`, which lives in the `prism-data`
+      |bucket within the `deployTools` account.
```

Comment:
This level of detail is excellent, though given it's not available on other deployment types, I wonder if it should be here, as, for example, it's disclosing bucket names.

Reply:
That's a good point, and I thought about it before adding it to the docs. However, I wouldn't have figured this out if @philmcmahon hadn't told me about it - I couldn't find it in the Riff-Raff documentation. 
I'm just trying to think what would be a  good compromise: we don't want to disclose private bucket names, but at the same time we don't want people to read the docs and think "How do I add my service to Deployment Resources"? 
Do you think it's acceptable to write something along the lines of "If you're not sure how to do this, contact the DevX team"?

Reply:
I wonder if https://github.com/guardian/deploy-tools-platform/blob/main/cloudformation/riffraff/riff-raff-roles.md is the place for this information?

Reply:
It could well be, but I still worry about discoverability. I had no idea that repo existed until today 😨 
I'll just remove the reference to private buckets for now - I think the "Add your App to Deployment Resources" is probably good enough in the sense that it provides at the very least some direction.

Reply:
@philmcmahon pointed out I could link to that file - I had some reservations about linking to a private repo from a public one but imho it's better than not saying anything!